flang/include/flang/Lower/PFTBuilder.h

```diff
@@ -247,6 +247,25 @@ struct Evaluation : EvaluationVariant {
     }});
   }
 
+  EvaluationList &getNestedEvaluations() {
+    assert(evaluationList && "no nested evaluations");
+    return *evaluationList;
+  }
+
+  bool hasAtLeastOneNestedEvaluation() const {
```

Comment:
Maybe `hasNestedEvaluations` is a little better?  (Your choice; an argument can be made for `hasAtLeastOneNestedEvaluation`.)

Reply:
Sure, its nicer, I'll go for it.